daemon-scheduler/pkg/deployment/deployment_worker.go

```diff
@@ -134,6 +159,10 @@ func (d deploymentWorker) updateDeployment(environment *types.Environment,
 }
 
 func (d deploymentWorker) deploymentCompleted(tasks []*ecs.Task, failures []*ecs.Failure) bool {
+	if len(tasks) == 0 {
```

Comment:
So a deployment can be completed but can have new sub-deployments later? Like when a new instance is added to the cluster?